NumLib/DOF/MeshComponentMap.h

```diff
@@ -45,9 +45,24 @@ class MeshComponentMap final
     /// The order (BY_LOCATION/BY_COMPONENT) of components is the same as of the
     /// current map.
     ///
+    /// This is single-component version.
+    ///
     /// \param component_id  The global components id.
     /// \param components components that should remain in the created subset
-    MeshComponentMap getSubset(std::size_t const component_id,
+    MeshComponentMap getSubset(std::size_t const component_ids,
+                               MeshLib::MeshSubsets const& components) const;
+
+    /// Creates a multi-component subset of the current mesh component map.
+    /// The order (BY_LOCATION/BY_COMPONENT) of components is the same as of the
+    /// current map.
+    ///
+    /// This is multi-component version.
+    ///
+    /// \note For each component the same mesh subset will be used.
```

Comment:
OK, but actually I wanted to say that the docu should use the exact class name, either `MeshSubset` or `MeshSubsets`, not "mesh subset".